packages/table/stories/Table.stories.tsx

```diff
@@ -41,7 +41,7 @@ const cityCellRenderer = ({ city }: { city?: string }) => (
 );
 const roleCellRenderer = ({ role }: { role?: string }) => (
   <Cell>
-    <em>{role}</em>
+    <span>{role}</span>
```

Comment:
Why change this? The example was intended to show that the user can define how the contents of a cell looks like.